### ISSUE TITLE
网络图片出不来解决.

### ISSUE THREAD

**Issue by jocongmin** [kuckboy1994/mp_canvas_drawer#58]
遇到网络图片出不来的问题，这个要么图片链接有问题，有么就是代码有问题，然后我遇到的是图片链接有问题，首先图片链接要https才行，然后图片服务器的域名要添加到微信的小程序后台的downloadfile合法域名中，这样才能正常的加载出来。

**Comment by jocongmin** [comment]
补充图片不能用base64 因为微信的文件下载不支持base64

**Comment by zmGitHub** [comment]
是的 真机测试之前一定要验证下图片的链接是不是支持 https 有没有在小程序后台的downloadfile合法域名中, 可以这样验证问题: 加载本地图片测试一次, 然后再加载网络图片测试一次 就清楚了
PS: 真机打开调式或者真机调式模式 即使图片路径不在downloadfile合法域名也是可以的
